Fluid/Filters/MiscFilters.cs

```diff
@@ -97,5 +101,165 @@ public static FluidValue EscapeOnce(FluidValue input, FilterArguments arguments,
             return new StringValue(WebUtility.HtmlEncode(WebUtility.HtmlDecode(input.ToStringValue())));
         }
 
+        public static FluidValue Date(FluidValue input, FilterArguments arguments, TemplateContext context)
+        {
+            DateTimeOffset value;
+
+            if (input.Type == FluidValues.String && input.ToStringValue() == "now")
```

Comment:
It would be nice if we use
```csharp
private static readonly string _now = "now"; 
```
if string casing is required, otherwise we should use `Eqauls()` instead